src/core/SupportingDocument.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ISupportingDocument,
+  SupportingDocumentCategory,
+  SupportingDocumentPlugin,
+  SupportingFile,
+  SupportingFileType,
+} from '../lib';
+import { Document } from './Document';
+
+/**
+ * Supporting document instance
+ * Cf: https://developers.algoan.com/api/#operation/getSupportingDocuments
+ */
+export class SupportingDocument extends Document implements ISupportingDocument {
+  /**
+   * Category of the supporting document
+   */
+  public category: SupportingDocumentCategory;
+
+  /**
+   * If not defined, it takes all values for a given category by default.
+   */
+  public validFileTypes: SupportingFileType[];
+
+  public files: SupportingFile[];
+
+  /**
+   * List of plug-ins linked to the supporting-document process
+   */
+  public plugIn: SupportingDocumentPlugin;
+
+  public readonly apiPath: string;
```

Comment:
Please add a little description if it is a public variable

Reply:
In fact, this property is useless. So I remove it.